client/components/Chat.vue

```diff
@@ -1,16 +1,18 @@
 <template>
-	<div id="chat-container" class="window" :data-current-channel="channel.name">
-		<div
-			id="chat"
-			:class="{
-				'hide-motd': !$store.state.settings.motd,
-				'colored-nicks': $store.state.settings.coloredNicks,
-				'show-seconds': $store.state.settings.showSeconds,
-			}"
-		>
+	<div
+		id="chat-container"
+		:class="{
+			window: true,
+			'hide-motd': !$store.state.settings.motd,
+			'colored-nicks': $store.state.settings.coloredNicks,
+			'show-seconds': $store.state.settings.showSeconds,
+		}"
+		:data-current-channel="channel.name"
+	>
+		<div id="chat">
 			<div
 				:id="'chan-' + channel.id"
-				:class="[channel.type, 'chan', 'active']"
+				:class="[channel.type, 'chan', 'chat-view']"
```

Comment:
make `channel.type` a data attribute?

Reply:
Yeah, will do. `.chan` should no longer be needed after I'm done with this.